[Fixes: #12140] Don't show gap after 30 days and allow user to set ms

cammellos opened the pull request:
This commit adds a setting in sync section so that user can select the
amount of days the can sync.

status-im-auto commented:
### Jenkins Builds
| :grey_question: | Commit | :hash: | Finished (UTC) | Duration | Platform | Result |
|-|-|-|-|-|-|-|
| :x: | e5d7cb91 | [#1](https://ci.status.im/job/status-react/job/prs/job/android/job/PR-12166/1/) | 2021-05-28 16:11:58 | ~16 sec | `android` | [:page_facing_up:`log`](https://ci.status.im/job/status-react/job/prs/job/android/job/PR-12166/1/consoleText) |

flexsurfer commented on src/status_im/ui/screens/default_sync_period_settings/view.cljs:
```diff
@@ -0,0 +1,30 @@
+(views/defview default-sync-period-settings []
+  (views/letsubs [{:keys [default-sync-period]} [:multiaccount]]
+    [react/view {:flex 1}
+     [topbar/topbar {:title (i18n/label :t/show-profile-pictures)}]
```
show-profile-pictures ?

flexsurfer reviewed (approved):
did you write py test ? :)

status-im-auto commented:
## 86% of end-end tests have passed
```
Total executed tests: 70
Failed tests: 10
Passed tests: 60
```
```
IDs of failed tests: 6291,6210,5314,5412,6328,5758,6645,5360,6263,6249 
```
<h3>Failed tests (10)</h3><details><summary>Click to expand</summary><br/><li><a href="https://ci.status.im/job/end-to-end-tests/job/status-app-prs-rerun/parambuild/?BRANCH_NAME=develop&APK_NAME=12166.apk&PR_ID=12166&TR_CASE_IDS=6291,6210,5314,5412,6328,5758,6645,5360,6263,6249">Rerun tests</a></li><br/><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2243654">test_keycard_can_see_all_transactions_in_history</a>, id: 6291 </b></td></tr><tr><td><p><blockquote><div>Device 1: *Find TransactionHistoryButton by accessibility id:* `History-item-button` </div><div>Device 1: *Tap on found TransactionHistoryButton* </div></blockquote></p><code> Device 1:  TransactionElement by xpath:`//android.view.ViewGroup[@content-desc="transaction-item"]` is not found on the screen</code><br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/c8ebe16e2cc64068962119cbeffb94d1?auth=82f43dfaa2789bee9e88c5f11d1cf1e4">Steps, video, logs</a></li><li><a href="https://assets.saucelabs.com/jobs/c8ebe16e2cc64068962119cbeffb94d1/0030screenshot.png?auth=82f43dfaa2789bee9e88c5f11d1cf1e4">Failure screenshot</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2243637">test_open_blocked_secure_not_secure_inlalid_offline_urls</a>, id: 6210 </b></td></tr><tr><td><p><blockquote><div>Device 1: *Tap on found Button* </div><div>Device 1: *Find Button by xpath:* `//*[@text="Continue anyway"]` </div></blockquote></p><code> Device 1: Button by xpath: `//*[@text="Continue anyway"]` is not found on the screen</code><br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/50bbbd7ed7e5462db77571a9cc7fbf75?auth=d232dc379dafe4651dbb9ba82e369661">Steps, video, logs</a></li><li><a href="https://assets.saucelabs.com/jobs/50bbbd7ed7e5462db77571a9cc7fbf75/0023screenshot.png?auth=d232dc379dafe4651dbb9ba82e369661">Failure screenshot</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2243602">test_can_see_balance_and_all_transactions_history_on_cellular</a>, id: 5314 </b></td></tr><tr><td><p><blockquote><div>Device 1: *Tap on found TransactionHistoryButton* </div><div>Device 1: *Pull to refresh view* </div></blockquote></p><code> Device 1:  TransactionElement by xpath:`//android.view.ViewGroup[@content-desc="transaction-item"]` is not found on the screen</code><br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/0c616b470ce9481dab87c979dc93e63d?auth=da89a231d502745696097170408b74f8">Steps, video, logs</a></li><li><a href="https://assets.saucelabs.com/jobs/0c616b470ce9481dab87c979dc93e63d/0027screenshot.png?auth=da89a231d502745696097170408b74f8">Failure screenshot</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2243625">test_insufficient_funds_wallet_positive_balance</a>, id: 5412 </b></td></tr><tr><td><p><blockquote><div>Device 1: *Getting STT amount* </div><div>Device 1: *Scrolling down to SilentButton* </div></blockquote></p><code> Device 1: SilentButton by xpath:* `//android.view.ViewGroup[@content-desc=':STT-asset-value']//android.widget.TextView[1]` is not found on the screen</code><br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/c0920186e97f48168af8ba93817874af?auth=2cb782d8e9480612f752b9fe55d5990e">Steps, video, logs</a></li><li><a href="https://assets.saucelabs.com/jobs/c0920186e97f48168af8ba93817874af/0025screenshot.png?auth=2cb782d8e9480612f752b9fe55d5990e">Failure screenshot</a></li></ul></p></ul></p></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2243660">test_send_transaction_set_recipient_options</a>, id: 6328 </b></td></tr><tr><td><p><blockquote><div>Device 1: *Tap on found Button* </div><div>Device 1: *Find Button by xpath:* `//*[@text="↓ 1000 MDS"]` </div></blockquote></p><code> Device 1: Button by xpath: `//*[@text="↓ 1000 MDS"]` is not found on the screen</code><br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/22e241d9d0754c67857f5296ff7f1886?auth=93a7023e79c6b56429e7d2dd4d9c7242">Steps, video, logs</a></li><li><a href="https://assets.saucelabs.com/jobs/22e241d9d0754c67857f5296ff7f1886/0039screenshot.png?auth=93a7023e79c6b56429e7d2dd4d9c7242">Failure screenshot</a></li></ul></p></ul></p></td></tr><tr><td><b>6. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2243631">test_keycard_can_recover_keycard_account_card_pairing</a>, id: 5758 </b></td></tr><tr><td><p><blockquote><div>Device 1: *Getting LXS amount* </div><div>Device 1: *Scrolling down to SilentButton* </div></blockquote></p><code> Device 1: SilentButton by xpath:* `//android.view.ViewGroup[@content-desc=':LXS-asset-value']//android.widget.TextView[1]` is not found on the screen</code><br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/b8679c869a3141e1840ee373ee101dcf?auth=ec2404cf2efeb8a2521a64cab492044e">Steps, video, logs</a></li><li><a href="https://assets.saucelabs.com/jobs/b8679c869a3141e1840ee373ee101dcf/0027screenshot.png?auth=ec2404cf2efeb8a2521a64cab492044e">Failure screenshot</a></li></ul></p></ul></p></td></tr><tr><td><b>7. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2243662">test_restore_account_migrate_multiaccount_to_keycard</a>, id: 6645 </b></td></tr><tr><td><p><blockquote><div>Device 1: *Getting ADI amount* </div><div>Device 1: *Scrolling down to SilentButton* </div></blockquote></p><code> Device 1: SilentButton by xpath:* `//android.view.ViewGroup[@content-desc=':ADI-asset-value']//android.widget.TextView[1]` is not found on the screen</code><br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/13494a0f78764c208d10478f1661a112?auth=ca00bac87561b2954649a5b0d089853b">Steps, video, logs</a></li><li><a href="https://assets.saucelabs.com/jobs/13494a0f78764c208d10478f1661a112/0063screenshot.png?auth=ca00bac87561b2954649a5b0d089853b">Failure screenshot</a></li></ul></p></ul></p></td></tr><tr><td><b>8. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2243614">test_unread_messages_counter_public_chat</a>, id: 5360 </b></td></tr><tr><td><p><blockquote><div>Device 2: **Signed in successfully!** </div><div>Device 2: **Looking for chat '#sfvjagt'** </div></blockquote></p><code> Counter is not shown for mention in public chat</code><br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/313039e24944441da04e1b6bee6a4920?auth=ce477bc670f731b7240f8e895efe37b1">Steps, video, logs</a></li><li><a href="https://assets.saucelabs.com/jobs/313039e24944441da04e1b6bee6a4920/0025screenshot.png?auth=ce477bc670f731b7240f8e895efe37b1">Failure screenshot</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/515e0df047f14fd9954e997d0f6202b9?auth=e61ce009bcbda58a569ec3af53672264">Steps, video, logs</a></li><li><a href="https://assets.saucelabs.com/jobs/515e0df047f14fd9954e997d0f6202b9/0023screenshot.png?auth=e61ce009bcbda58a569ec3af53672264">Failure screenshot</a></li></ul></p></ul></p></td></tr><tr><td><b>9. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2243647">test_request_and_receive_stt_in_1_1_chat_offline</a>, id: 6263 </b></td></tr><tr><td><p><blockquote><div>Device 2: *Element TransactionStatus text is equal to Confirmed* </div><div>Device 1: *Wait for text element TransactionStatus to be equal to Confirmed* </div></blockquote></p><code> Device 1: 'Pending' is not equal to expected 'Confirmed' in 60 sec</code><br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/5b7334a9961c4d56ac00ce3c2a106815?auth=f3329a94e65ab9f3cc7f6cd7be25a2c4">Steps, video, logs</a></li><li><a href="https://assets.saucelabs.com/jobs/5b7334a9961c4d56ac00ce3c2a106815/0030screenshot.png?auth=f3329a94e65ab9f3cc7f6cd7be25a2c4">Failure screenshot</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/456368f282434c8b923ee8cc886f5679?auth=f0427de340eb5b56251f4f78ff76fd5d">Steps, video, logs</a></li><li><a href="https://assets.saucelabs.com/jobs/456368f282434c8b923ee8cc886f5679/0047screenshot.png?auth=f0427de340eb5b56251f4f78ff76fd5d">Failure screenshot</a></li></ul></p></ul></p></td></tr><tr><td><b>10. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2243644">test_keycard_request_stt_from_daap</a>, id: 6249 </b></td></tr><tr><td><p><blockquote><div>Device 1: *Getting STT amount* </div><div>Device 1: *Scrolling down to SilentButton* </div></blockquote></p><code> Device 1: SilentButton by xpath:* `//android.view.ViewGroup[@content-desc=':STT-asset-value']//android.widget.TextView[1]` is not found on the screen</code><br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/4f4168f465564f9c997817a4d6224018?auth=9898a651a4e1e4befef1d33488493bdc">Steps, video, logs</a></li><li><a href="https://assets.saucelabs.com/jobs/4f4168f465564f9c997817a4d6224018/0037screenshot.png?auth=9898a651a4e1e4befef1d33488493bdc">Failure screenshot</a></li></ul></p></ul></p></td></tr></tbody></table></details><h3>Passed tests (60)</h3><details><summary>Click to expand</summary><br/><br/><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2243610">test_sign_message_and_transactions_filters_from_daap</a>, id: 5342 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/cd7307d853f6423883d4475243c2d4c7?auth=a40dc1a0e4a2e212a4feb4e085f186da">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2243657">test_image_in_one_to_one_send_save_reply_timeline</a>, id: 6305 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/39e999a6ad09488b845aed6b0f4d0f74?auth=c8510d73e7a06dfaf3f4f9993e0c09d6">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/7a85036bf41e4950b4de3cb12315d4eb?auth=af9e55afc908dbe71bcb50a118ff7b57">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2243638">test_add_account_to_multiaccount_instance_generate_new</a>, id: 6224 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/acdfee67493143e488a9cd5556e70f5b?auth=cb543f7f534cb097bd3a09b9e2c6fdf8">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2243643">test_add_and_delete_watch_only_account_to_multiaccount_instance</a>, id: 6244 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/e4b3a643fb764ea0a3b43db46bab4f43?auth=3828df1fb23f6d55e2e5f6662d83af0c">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2243623">test_open_public_chat_using_deep_link</a>, id: 5396 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/0e107423b1a2402a93feecd3df1730cb?auth=605047fe81ce964b914e2aa23f4816b0">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>6. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2243596">test_offline_add_new_group_chat_member</a>, id: 3998 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/43176083fed1400da54a43063d4a516c?auth=019de8368c5fc3dc1052941707cacdd2">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/6c9723b6209b4eeeb5527a986ed75630?auth=5607da83e59fefc0d7aa5731a1d39af5">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://saucelabs.com/jobs/a7e7f428e29a40e7aabeae1e65ca2e56?auth=14bfe330b3815f7098aac33efa0c4749">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>7. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2243603">test_send_non_english_message_to_newly_added_contact_on_different_networks</a>, id: 5315 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/b5fb99b71b924714935d5dfc1068f8b6?auth=2ed96370c9ff6ecd1271391aa18d88c0">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/250f58f2a2ae46af9cdb92f9d9f2b9b4?auth=d09ce7207d129752faaca342f509389c">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>8. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2243629">test_keycard_create_login_resotore_unlock_same_seed</a>, id: 5689 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/964d7f17786846c0aae2e3beb2c62a29?auth=1bc5e2a640a727736a9fd818f0442e44">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/8756b2abb43749f18141a94138248325?auth=2ba6b8bd5d4fd6b2cce4a9ebe32a9ac9">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>9. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2243621">test_delete_close_all_tabs</a>, id: 5390 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/ed9d7d4d1ad7453cad5e05ee613bb374?auth=72602e47ab4c4f7c3f34599e330d92f3">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>10. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2243595">test_create_new_group_chat_messaging_pn_delivered</a>, id: 3994 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/6deb606d78e74742b148738285ab41fe?auth=e84cacf82108ef3aaa499c64189f8471">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/70ea3842949542b089f5eb9491663106?auth=45a54d396a48d1d9fc28da4829baca50">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>11. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2243612">test_send_token_with_7_decimals</a>, id: 5350 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/143a3d8108b146748c36cabe6a4df835?auth=69f53eadf2caf57fb080f5601ebd5cc1">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>12. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2243620">test_delete_chats_via_delete_button_rejoin</a>, id: 5387 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/68be5d7f40c04d8e8682fe026713702c?auth=96cf0d35c5058bd3ab082617c7ffc561">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>13. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2243645">test_keycard_sign_message_and_transactions_from_daap</a>, id: 6251 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/75870a15fcee48388e5ab7c30e18d683?auth=1869799dd816933017b73a8eec7f7e00">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>14. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2243656">test_recover_account_from_new_user_seedphrase</a>, id: 6296 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/58566feb7ca24ace8ae4bba2711eaed5?auth=cbefc5ecd4a1e1734064d626ea635153">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>15. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2243635">test_block_user_from_public_chat</a>, id: 5786 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/d8c9b5a70c8440ac8ed7da03424b02fc?auth=aca6af5d279096f059936b03f85da461">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/2e546f615ecb41518d8a934e01a158e7?auth=b4acc6245bd7ecab7bedf455bb3dbf5f">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>16. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2243659">test_send_audio_message_with_push_notification_check</a>, id: 6316 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/bdf21557416240ca9dfb19ec021be83f?auth=f31fa18b844b197746b6bc109e44acfd">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/be3cc067d023471e8039116611446b47?auth=18437aadb7b3dbaaafcc880461287723">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>17. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2243628">test_redirect_to_public_chat_tapping_tag_message_fetch_more_history</a>, id: 5675 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/5477e982711d45569b2a2139280a913f?auth=932964f30bafd29dc4801622123fc3c0">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>18. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2243639">test_ens_mentions_pn_and_nickname_in_public_and_1_1_chats</a>, id: 6226 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/92352179be214ca58686d4f920825af7?auth=09eb7f019b76486e2973291fa10659d6">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/5d0ecfaf598a494ab7dcf8287a1d6010?auth=33b0855d6a7a1da4dfacc154d7b5000d">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>19. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2243652">test_keycard_send_eth_from_wallet_to_address</a>, id: 6289 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/83b410534adc4628a6031cf197e4d285?auth=5343d66e7088a6a0044c1713ac1669b4">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>20. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2243649">test_add_account_to_wallet_private_key_and_seed_phrase</a>, id: 6272 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/e9c1141f7cf04f66837fde7a87a0a6c8?auth=aafa6352a47c42d11fcb296e1a27db6a">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>21. test_back_forward_refresh_navigation_history_kept_after_relogin</b> (TestRail link is not found)</td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/22af5be1a90646b59e21cf28a05d75c5?auth=bc3c4b63bdd942cc75b73713c5dd3206">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>22. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2243600">test_offline_is_shown_messaging_1_1_chat_sent_delivered</a>, id: 5310 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/8d8b79d77fac4253833a12097ac738e5?auth=74a36669fbd989280af19fde52ba9cf1">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/b663c8d054354b999acb7e165177543f?auth=2907b18c58ebf7086c0a86122992e1a5">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>23. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2243632">test_pair_devices_sync_one_to_one_contacts_nicknames_public_chat</a>, id: 5762 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/bbceece12a364bb5a377c77958168f4e?auth=a5e7d921d19eaa6aa8656a4918bbd14c">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/11444062a5b5463abefb26708e2449c0?auth=f1a53667b027cdd2ef9551858201101b">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>24. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2243605">test_long_press_to_delete_chat</a>, id: 5319 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/edf104558df2480582e49ce575533c1a?auth=ede3ea18a493e86c840fc4e8057fb8d4">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>25. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2243604">test_copy_and_paste_messages</a>, id: 5317 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/1c70f3b5374346f3a6f411c0dbc2f9bd?auth=c15f61b3c08f02890398648b586ec882">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>26. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2243619">test_open_transaction_on_etherscan_copy_tx_hash</a>, id: 5384 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/0498ac16334a4fb18cb16d7cd1744109?auth=45ef2a7d3c4ca0dbc8d3c7be8d7e6cdc">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>27. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2243618">test_user_can_see_all_own_assets_after_account_recovering</a>, id: 5381 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/c5b105a7826141448245c8b256b8bd5b?auth=56f3df25581b0c7190a0751c14b643c2">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>28. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2243598">test_send_eth_from_wallet_to_address_incorrect_password</a>, id: 5308 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/c7a16d5e81a44968be554084830d5c4d?auth=3958dbec85a01a7f13fd23a492618ee5">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>29. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2243640">test_mobile_data_usage_complex_settings</a>, id: 6228 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/bd71e52f0cfb4d049143adc54fd03d03?auth=8814547c42e71c81020c1e2259e399ff">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/40c1055b2178481fb7844bee6262d588?auth=6709228f69737fa4bc76a817d55e7dee">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>30. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2243624">test_start_chat_with_ens_mention_in_one_to_one</a>, id: 5403 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/149eb3ed40fc4306a3a404e4657cc7ea?auth=50b7c7964e60865d64cc24b942a3d12f">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>31. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2243648">test_decline_transactions_in_1_1_chat_push_notification_changing_state</a>, id: 6265 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/138851c363584ee88e440870a8997d9a?auth=faad4db8e7278deead77f50215a22327">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/4f79dd79980749b0ab8f86e895c9b8f6?auth=6b236c2c5f624b02f592fb926ad66695">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>32. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2243646">test_send_eth_in_1_1_chat_transaction_push</a>, id: 6253 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/64388ad7c049482b854919c057594dfc?auth=719bb6249a71813a4e462b1aa5b327c8">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/5119b59dfa2649de99ffa0c9d062bb3f?auth=240a7f46a8150ede20ec11f18341ab11">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>33. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2243633">test_install_pack_and_send_sticker</a>, id: 5782 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/9b23eab6eeb147308d542cc8c52e9a46?auth=77cf12681dcef045899091a2f149923b">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/7747ff1189654af38bc7eb0c1dc673b0?auth=31c8cdcbafc155a83b5bc6ea33c60fe5">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>34. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2243613">test_switch_users_special_char_password_and_add_new_account_logcat</a>, id: 5356 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/2eafba5d28b04bab8f6f07b3b4174c35?auth=77f7fdfa780a006947ea2d55ea9e2c53">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>35. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2243609">test_manage_assets</a>, id: 5341 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/e5caabdf9cab4232ab38f6b2969ec5c9?auth=3c90d0c9ce2baf0ab00120be7af1b2b0">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>36. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2243661">test_browser_managing_bookmarks</a>, id: 6633 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/5f5e640606664207b013b36db448e160?auth=4403fa81033eb2ec681ff5486d5fd5ed">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>37. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2243597">test_open_chat_by_pasting_chat_key_check_invalid_chat_key_cases</a>, id: 5304 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/10c7cc120ed64020a9fca81359d56984?auth=d19b0f43491fc6eb1e67d0eebdefb8d8">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>38. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2243599">test_request_stt_from_daap</a>, id: 5309 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/e19253ea74fa48c698518294998dc564?auth=5aa0ccd23ce0421a6e7be3963acb7b2f">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>39. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2243634">test_can_use_purchased_stickers_on_recovered_account</a>, id: 5783 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/63d7b4c3e7e546c5a670f8aecef213d2?auth=19844c4ee01c2bd02eda6bbf00d2fb22">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>40. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2243630">test_dapps_permissions</a>, id: 5738 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/78a8998899614020a45653cbea6f82af?auth=4a839bad242958c080bb769243f90c75">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>41. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2243622">test_account_recovery_with_uppercase_whitespaces_seed_phrase_special_char_passw_logcat</a>, id: 5394 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/25f90624ed434839885a22d445f92665?auth=ae2be9449c0f8372003faae61b783662">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>42. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2243650">test_send_eth_to_ens_in_chat</a>, id: 6279 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/6156e73c8cfa4060966b1519a27de728?auth=14e30362bf0296710a4491df21410542">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>43. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2243606">test_share_copy_contact_code_and_wallet_address</a>, id: 5323 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/e1554f72832041e79451c3a00a358c82?auth=ca218e469a5877b7a8ff9e3839d6eb86">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>44. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2243608">test_wallet_set_up</a>, id: 5335 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/842672aacd8f4ed299aad70f1b1d1186?auth=882ff14aaab874b72c1fcc0e5a0ddcb9">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>45. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2243636">test_send_transaction_with_custom_token</a>, id: 6208 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/c47d7336112f4a70ab22de983798f333?auth=396c7c6030cb65823aabf01bc6911e5d">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>46. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2243617">test_home_view</a>, id: 5379 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/d49b91c62d4e4ddbbb7214b81802b227?auth=fc316a1974559735fbc47f588740c20c">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>47. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2243627">test_can_add_existing_ens</a>, id: 5502 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/ee8ca2a45056439ba61d6ca19efade79?auth=cb0f88ca8a7c32923337820fae12e732">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>48. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2243626">test_logcat_backup_recovery_phrase</a>, id: 5419 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/40ffcc23e86e42508aa55e4dddc2b0c5?auth=a8ba93f738b4ec7531c05049eccd2dcd">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>49. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2243663">test_set_profile_picture</a>, id: 6646 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/c04c4ff00f3a4b959b2a275fb7f109ee?auth=a4c6141439ae9bd941859270a441f85d">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/adc475ba4bb94297908c2118cfc815ab?auth=3cf187b21545adae853191ce64bed3a2">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>50. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2243611">test_collectible_from_wallet</a>, id: 5346 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/be9b1640d29c4e36942127326b0b6e88?auth=4d3cccfd64807e3c0f72a16aa2494b29">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>51. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2243607">test_add_and_remove_mention_contact_with_nickname_from_public_chat</a>, id: 5332 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/4bcf466a072045088c3a3ea58d4ddcf7?auth=a5798a8be8613f3c51d39521189c8908">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/c70b7c0f27354149842b0a5682126d41?auth=2de3370a26b18879fcecce87e34fdc4f">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>52. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2243642">test_restore_account_from_mnemonic_to_keycard</a>, id: 6240 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/c72985c14888447b9c2b70f7c289be07?auth=aabac5e0259b6a392853b9c01d9be77b">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>53. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2243615">test_pass_phrase_validation</a>, id: 5363 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/f45b9fc4698243e99cdfe3e9e5b9b98f?auth=7f956c5150a36cdfb5ceebd0910acdbc">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>54. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2243655">test_keycard_send_eth_in_1_1_chat</a>, id: 6293 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/f5cf6e5b27654881a60d57178925dad5?auth=692c91dae8e02ef3ae6a827edc3b48e1">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/90969ec6b11b4961b8c48304ee6b1c00?auth=37fb9e7aff692d1c02861f52b87e811b">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>55. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2243616">test_send_and_open_links_with_previews</a>, id: 5373 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/3ba5f9c473c64a55ac5fdf613aef6be3?auth=613edd5e73bf4fed734dd6d625fab171">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/9e29435a15ba482292442b78ebd3041d?auth=a0fd3ac225ce2bfc5c367a3f941508bc">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>56. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2243658">test_reactions_to_message_in_chats</a>, id: 6315 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/3d66b438d5594f55a76b9e9d37e461b5?auth=c0708d93f25e31e17f29882834d3265a">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/4952380a4127409190e5950718224d16?auth=b31ffd0adc16204aa0e8f1ccf9891ea2">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>57. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2243641">test_fetching_balance_after_offline</a>, id: 6237 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/66c24c9e9505426989ee40dad5bd3453?auth=60de7857bedf43aeddc739bc480f1481">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>58. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2243651">test_push_notification_1_1_chat_no_pn_activity_center</a>, id: 6283 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/0cfcac39b8274f5892fba8c73cd5c5d2?auth=4272f26871c81bb299222a5f08f4ad78">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/1a7a199d36a94b41a3f4c03b49f52c54?auth=8f8109b2bf9c2bd0fcbae20ac70f88b3">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>59. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2243601">test_public_chat_messaging_emojis_timestamps</a>, id: 5313 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/ac716cf212614d5c9259c5773b1c6ff1?auth=104283074f959d5ac9494b74101eaa4b">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/106406e5982a4574b001160e639a455f?auth=2f435c16fd86555bce45c31bbcb8e98b">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>60. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2243653">test_keycard_fetching_balance_after_offline</a>, id: 6290 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/f6ba23ed8b27495d8ddbfbd80e6849c7?auth=716e638d3fcf485dc180111aaf896462">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table></details>

cammellos committed:
[Fixes: #12140] Don't show gap after 30 days and allow user to set sync period

This commit adds a setting in sync section so that user can select the
amount of days the can sync.

Signed-off-by: Andrea Maria Piana <andrea.maria.piana@gmail.com>

status-im-auto commented:
## 90% of end-end tests have passed
```
Total executed tests: 10
Failed tests: 1
Passed tests: 9
```
```
IDs of failed tests: 5360 
```
<h3>Failed tests (1)</h3><details><summary>Click to expand</summary><br/><li><a href="https://ci.status.im/job/end-to-end-tests/job/status-app-prs-rerun/parambuild/?BRANCH_NAME=develop&APK_NAME=12166.apk&PR_ID=12166&TR_CASE_IDS=5360">Rerun tests</a></li><br/><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2244489">test_unread_messages_counter_public_chat</a>, id: 5360 </b></td></tr><tr><td><p><blockquote><div>Device 2: **Signed in successfully!** </div><div>Device 2: **Looking for chat '#wzkmvwh'** </div></blockquote></p><code> Counter is not shown for mention in public chat</code><br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/964cf40c448e403cbea5842b4002b31d?auth=cf1294661da6bab74e8b60b40034f2f4">Steps, video, logs</a></li><li><a href="https://assets.saucelabs.com/jobs/964cf40c448e403cbea5842b4002b31d/0025screenshot.png?auth=cf1294661da6bab74e8b60b40034f2f4">Failure screenshot</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/aebc8437e5c54d9ea350f2c6927658f1?auth=e20050868834366c3725066ab3cbfd10">Steps, video, logs</a></li><li><a href="https://assets.saucelabs.com/jobs/aebc8437e5c54d9ea350f2c6927658f1/0023screenshot.png?auth=e20050868834366c3725066ab3cbfd10">Failure screenshot</a></li></ul></p></ul></p></td></tr></tbody></table></details><h3>Passed tests (9)</h3><details><summary>Click to expand</summary><br/><br/><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2244497">test_restore_account_migrate_multiaccount_to_keycard</a>, id: 6645 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/dde9f99944f243edb60d98b27f60a8d7?auth=0690574f49ac09b0dd1f1a4990f7f989">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2244496">test_send_transaction_set_recipient_options</a>, id: 6328 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/3668c540f52d4f38b11b85740522f72c?auth=dc4e0080664b68a7d27e72ef1f6518e0">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2244493">test_keycard_request_stt_from_daap</a>, id: 6249 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/74d27f671143411198ae49d2aeada49b?auth=044d3677bfdacd33ecc3398618878619">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2244492">test_open_blocked_secure_not_secure_inlalid_offline_urls</a>, id: 6210 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/2ba26f209bf04cd29c4859fa4423d788?auth=6a33e707e7886a3a971393dc56c7e8d3">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2244495">test_keycard_can_see_all_transactions_in_history</a>, id: 6291 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/2152354a6077455d99a740a7f24f0277?auth=970dce4c054c4e6485dc855042e093fb">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>6. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2244494">test_request_and_receive_stt_in_1_1_chat_offline</a>, id: 6263 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/afa3f7c0cb0e4a18bfe70b10ea9860ae?auth=9a3b242b97157040f52eee2481c30b4d">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/72f3ace171554932a7d3833711362923?auth=d3102238f89da18b7126ea1b278304a7">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>7. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2244488">test_can_see_balance_and_all_transactions_history_on_cellular</a>, id: 5314 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/d54b4caadb0d4bbd9de309ddbc51f36e?auth=c1612ff7184ea76b8c975898dccc926e">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>8. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2244490">test_insufficient_funds_wallet_positive_balance</a>, id: 5412 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/a7413951086f4a6d9d66a481d509bb9f?auth=20f7694eaed2878980be4920842564ef">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>9. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2244491">test_keycard_can_recover_keycard_account_card_pairing</a>, id: 5758 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/74448ea3092e470c9691d18990bde999?auth=0d3e6840b8e0c952133ceca5d1ccd4b0">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table></details>
